locales/ta_IN/LC_MESSAGES/duckduckgo.po

```diff
@@ -163,43 +163,43 @@ msgid "About DuckDuckGo"
 msgstr "டக்டக்கோ-வைப் பற்றி'"
 
 msgid "About Us"
-msgstr ""
+msgstr "எம்மைப் பற்றி"
 
 #. Ad like 'advertisement'
 msgid "Ad"
 msgstr "விளம்பரம்"
 
 msgctxt "feedback form"
 msgid "Ad is annoying"
-msgstr ""
+msgstr "விளம்பரம் எரிச்சலூட்டுகிறது"
 
 msgctxt "feedback form"
 msgid "Ad is inappropriate"
-msgstr ""
+msgstr "பொருத்தமற்ற விளம்பரம்"
 
 msgctxt "feedback form"
 msgid "Ad is irrelevant"
-msgstr ""
+msgstr "சம்பந்தமற்ற விளம்பரம்"
 
 msgctxt "feedback form"
 msgid "Ad is malware"
-msgstr ""
+msgstr "தீங்குநிரல் கொண்ட விளம்பரம்"
 
 msgctxt "feedback form"
 msgid "Ad is suspicious"
-msgstr ""
+msgstr "சந்தேகத்திற்குரிய விளம்பரம்"
 
 #  Marked fuzzy because of translated trademark
 #. From Home page, options to "Add DuckDuckGo" to browser or set as home page.
 #, fuzzy
 msgid "Add DuckDuckGo"
-msgstr "சேர் கேலன் இலிருந்து வழிமாற்றப்பட்டது"
+msgstr "டக்டக்கோவை சேர்க"
 
 msgid "Add DuckDuckGo as a search engine"
-msgstr ""
+msgstr "டக்டக்கோவை உங்கள் முதன்மை தேடுபொறியாக அமைக்க"
 
 msgid "Add DuckDuckGo to %s"
-msgstr ""
+msgstr "டக்டக்கோவை %sல் சேர்க"
```

Comment:
Please, leave the brand name in English and in latin alphabet.

Reply:
Sure will do